aptos-move/framework/supra-framework/sources/poel/poel.move

```diff
@@ -0,0 +1,308 @@
+    public fun calculate_collaterisation_rate(
+        _asset_nominal_value: u64,
+        _total_nominal_value: u64,
+        _asset_id: u64
+    ) {
+
+    }
+
+
+    fun sign(_value: u64): u64 {
+        0
+    }
+
+
+    public fun set_parameters(
+        _account: &signer,
+        _coefficient_b: u64,
+        _coefficient_X: u64,
+        _coefficient_rho: u64,
+        _min_collateralisation: u64
+    ) {
+
+    }
+
+    /// Unlocks token from the all delegation pools involved in the system 
+    public fun unlock_tokens(_supra_amount: u64) {}
+
+    /// Delegates  token to the all delegation pools involved in the system 
+    public fun delegate_tokens(_supra_amount: u64) {}
+
+    /// Facilitates the withdrawal of unlocked tokens from the inactive delegation pools.
+    public fun withdraw_tokens() {}
+
+    /// This function facilitates the replacement of an existing staking pool with a new pool address.
+    /// It performs several checks and updates to ensure the integrity and security of the staking \
+    /// process within the blockchain network.
+    public fun allocate_rewards() {}
+
+
+    /// This function is designed to manage the transition of assets between staking pools. 
+    /// It ensures that the transition adheres to the required permissions
+    /// and conditions set within the system before proceeding with asset reallocation.
+    public fun replace_staking_pool (
+        _account: signer,
+        _replaced_pool_address: address,
+        _replacing_pool_address: address
+    ) {}
+
+    /// pdates asset prices and calculates new supply metrics for collateral management in the system.
+    public fun update_asset_price_supply() {}
+
+    ///  This function recalculates and updates(delegates or unlocks) the total amount of
+    /// Supra that is borrowed based on the current asset prices, supplies, and collateralization rates.
+    public fun update_rented_amount() {}
+
+    /// Enables increasing the rentable Supra amount by transferring funds to the PoEL contract.
+    public fun increase_rentable_amount(_account: &signer, _amount: u64) {}
+
+    /// Decreases the amount of rentable tokens available in the PoEL contract.
+    public fun decrease_rentable_amount(_acccount: &signer, _amount: u64) {}
+
+    /// Facilitates the creation of borrow requests following the deposition of the original asset into
+    /// an intermediary vault. One of the main reasons why the borrow_request function has been suggested 
+    /// in the flow to borrow is because the pending_active coins do not earn rewards. 
+    public fun borrow_request(
+        _account: &signer,
+        _asset_id: u64,
+        _asset_amount: u64,
+        _receiver_address: address
+    ) {}
+
+    /// Facilitates borrowing of assets, calculates new collateralization rates and manages distribution across staking pools.
+    public fun borrow(
+        _asset_id: u64,
+        _asset_amount: u64,
+        _receiver_address: address
+    ) {}
```

Comment:
Do not use "borrow" which has different connotation in Move and can be confusing for reader

Reply:
Okay, will change this to borrow_supra()

Reply:
okay. will change this to read borrow_supra()